docs/content/en/docs/Cluster operator/quickstart.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Quick start"
+linkTitle: "Quick start"
+description: >
+  The easest way to get start with Kurator cluster operator.
+---
+
+In this tutorial we’ll cover the basics of how to use Cluster API and Kurator cluster operator to create one or more Kubernetes clusters.
+
+## Setup kubernetes clusters with Kind
+
+Deploy a kubernetes cluster using kurator's scripts. This script will create three clusters for you, one is used to host Karmada control plane and the other two will be joined as member clusters.
+
+```bash
+git clone https://github.com/kurator-dev/kurator.git
+cd kurator
+hack/local-dev-setup.sh
+```
+
+## Build `cluster-operator` from source
+
+Build cluster operator image:
+
+```console
+make docker
+```
+
+Build cluster operator helm chart:
+
+```console
+make gen-chart
+```
+
+Load image to kind cluster:
+
+```console
+kind load docker-image ghcr.io/kurator-dev/cluster-operator:latest --name kurator
+```
+
+## Install cert manager
+
+Kurator cluster operator depends on [cert manager CA injector](https://cert-manager.io/docs/concepts/ca-injector).
+
+***Please make sure cert manager is ready before install cluster operator***
+
+```console
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+kubectl create namespace cert-manager
+helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true
+```
+
+## Install cluster operator
+
+```console
+# Please make sure cert manager is ready before install cluster operator
+kubectl create namespace kurator-system
+helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
+helm install -f <your_custom_values>.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
+```
+
+## Create a vanilla cluster on AWS
+
+The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
+
+```console
+# download clusterctl
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.5/clusterctl-linux-amd64 -o clusterctl
+sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
+# verify version
+clusterctl version
+```
+
+Create a kubernetes cluster on AWS, which contains control plane node and worker node, the cluster topology shows as following:
+
+{{< image width="75%"
+    link="./image/capa-aws.png"
+    >}}
+
+```console
+kubectl apply -f manifests/examples/capi-quickstart.yaml
```

Comment:
Can you add some desc about how this yaml come out

Reply:
it's generated by `clusterctl generate`, but the tools will not work with our deploy mode.